Config.ts

```diff
@@ -1,4 +1,4 @@
 export default {
-  network: process.env.REACT_APP_NETWORK ?? 'fuse',
+  network: process.env.REACT_APP_NETWORK ?? '66757365',
```

Comment:
created separate config file for network variable which is used for generating mnid in generatePaymentLink.ts

Reply:
i know what that is, i mean what is this strange default 6675... , defaults should be valid settings

Reply:
this is not a react app envs should not be prefiex with REACT_APP
also since this will be used as an sdk ie devs will import this package i'm not sure env variables can be used at all
@TomasGonzalez

Reply:
Oh yeah sorry for that. I was testing something through this.